### ISSUE TITLE
No way to acknowledge persistent alarm - Dash

### ISSUE THREAD

**Issue by ToreBj** [nightscout/AndroidAPS#903]
I had my Dash pod running for the extended period (8h) until it gave a continuous beeping tone, i.e. expired.
I deactivated the pod, everything was reset and ready for init of a new pod.
I activated the new pod with no problems.
However now I started to hear an increasing alarm sound, but could not see any notification to acknowledge.

I did the Dash pod change at 17:46. 10-nov-2021.

Restarting AAPS made no difference. I finally force stopped AAPS and the alarm was reset.

 
![image](https://user-images.githubusercontent.com/23278852/141265112-5cfbd91c-4882-4591-a2f2-2332f7612f1c.png)

[AndroidAPS.log](https://github.com/nightscout/AndroidAPS/files/7518947/AndroidAPS.log)

I'm using a Oneplus 7 Pro with Android v10. 
Dexcom G6 patched app, Xdrip+

**Comment by kristinsmotz** [comment]
Sounds similar to what I had in this issue: https://github.com/nightscout/AndroidAPS/issues/816
In that case the old pod failed during deactivarion, and a two-tone alert kept sounding. I tried to exit app, but no help, so finished activating new pod then shut down the phone to get rid of it. I think I saw a "null" error message in system notification at some point, but swiping it didn't kill the alert.
